src/components/TaskList.tsx

```diff
@@ -43,12 +55,12 @@ function ExpandableTaskItem({ item }: { item: TaskItem }) {
   );
 
   return (
-    <div key={item.id}>
+    <div key={userTask.Task.id}>
       <div className="flex bg-gray-50 border-b center">
         <input
           checked={isChecked}
           className="mr-2 mt-3 ml-2 h-6 w-6 inline-block"
-          id={item.id}
+          id={userTask.Task.id.toString()}
```

Comment:
why do we need this id? can we remove it?

Reply:
We can. I believe it was used only for the old local storage logic.

Thanks!

Reply:
> why do we need this id? can we remove it?

I'll be resolving this conversarion so I can merge. Ok?